examples/nlp/neural_machine_translation_with_keras_nlp.py

```diff
@@ -449,70 +455,6 @@ def token_probability_fn(decoder_input_tokens):
     print(translated)
     print()
 
-"""
-After 10 epochs, we get samples like these:
-
-** Example 0 **
-
-have you seen this?
```

Comment:
should we keep a few examples in? Will help readability when not running this as a colab.